sys-libs/kpmcore/kpmcore-4.0.1.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+FRAMEWORKS_MINIMAL="5.56"
```

Comment:
Please drop this, eclass already depends on 5.60 as a minimum. If you copy from kde overlay, you would ensure to get the most recent state that already got rid of this line.